go/types/indexed_sequences.go

```diff
@@ -77,3 +80,74 @@ func orderedKeyFromSum(msd []metaTuple) orderedKey {
 	}
 	return orderedKeyFromUint64(sum)
 }
+
+func LoadLeafSequences(c Collection, startIdx, endIdx uint64) (leaves []sequence, firstIdx uint64) {
```

Comment:
it would be useful to comment on inclusivity of startIdx/endIdx and what firstIdx means.
From reading the code it looks like it's inclusive of startIdx and exclusive of endIdx.